package.json

```diff
@@ -95,6 +95,11 @@
           "default": "# ${noteName}\n\n",
           "description": "Template for auto-created note files. Available tokens: ${noteName}, ${timestamp}. Timestamp is inserted in ISO format, i.e. 2020-07-09T05:29:00.541Z."
         },
+        "vscodeMarkdownNotes.compileSuggestionDetails": {
```

Comment:
What are the cases where someone would set this to false? If they just use `plaintext` notes?

Is there a perf implication?

Also, can you explain a little of your thinking around the `false` default? Generally that is my bias, but curious to hear both sides of this and could imagine this being true if there is not a perf reason and you think most people would want this on

Reply:
I forgot to change this default.

the fact is that I don't really like the display of markdownString, the size of the title is really unattractive, so I added this configuration, and if anyone who doesn't like this,  can also choose to set this option to false, but the default value should really be `true`!

![Screen Shot 2020-08-21 at 4 01 55 PM](https://user-images.githubusercontent.com/24939578/90867513-13994e80-e3c8-11ea-9bb3-a98767796a2d.png)
![Screen Shot 2020-08-21 at 4 02 54 PM](https://user-images.githubusercontent.com/24939578/90867523-16943f00-e3c8-11ea-86e8-29b7bd0cc0b7.png)